iMovie/Views/RatingCircleView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  RatingCircleView.swift
+//  iMovie
+//
+//  Created by MorsyElsokary on 04/01/2022.
+//
+
+import SwiftUI
+
+struct RatingCircle: View {
+    
+    // MARK: - Properties
+    
+    @State private var isLoading = false
+    @State private var progress: CGFloat = 0.86
```

Comment:
Do not add a default value for progress, we need to inject it from outside and I think no need for it to be `@State`